src/components/OSCALCatalogGroups.js

```diff
@@ -0,0 +1,91 @@
+import Box from "@mui/material/Box";
+import List from "@mui/material/List";
+import Paper from "@mui/material/Paper";
+import { styled } from "@mui/material/styles";
+import Tab from "@mui/material/Tab";
+import Tabs from "@mui/material/Tabs";
+import PropTypes from "prop-types";
+import React from "react";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
+import OSCALCatalogGroup from "./OSCALCatalogGroup";
+
+export const OSCALControlList = styled(List)`
+  padding-left: 2em;
+  padding-right: 2em;
+`;
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`vertical-tabpanel-${index}`}
+      aria-labelledby={`vertical-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
+    </div>
+  );
+}
+
+const ComponentTabs = styled(Tabs)(
+  ({ theme }) => `
+  border-right: 1px solid ${theme.palette.divider};
+  width: 15em;
+  min-width: 15em;
+`
+);
+
+function a11yProps(index) {
+  return {
+    id: `vertical-tab-${index}`,
+    "aria-controls": `vertical-tabpanel-${index}`,
+  };
+}
+
+const CatalogGroupTabButton = styled(Tab)`
+  width: 15em;
+  min-width: 15em;
+`;
+
+TabPanel.propTypes = {
+  index: PropTypes.number.isRequired,
+  value: PropTypes.number.isRequired,
+};
+
+export default function OSCALCatalogGroups(props) {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <OSCALSection>
+      <OSCALSectionHeader>Control Groups</OSCALSectionHeader>
+      <Paper sx={{ flexGrow: 1, bgcolor: "background.paper", display: "flex" }}>
```

Comment:
```suggestion
      <Paper sx={{ flexGrow: 1, display: "flex" }}>
```

Do we really need to set the background color?